src/vs/workbench/browser/parts/editor/editorPicker.ts

```diff
@@ -77,16 +78,22 @@ export class EditorPickerEntry extends QuickOpenEntryGroup {
 
 	public run(mode: Mode, context: IEntryRunContext): boolean {
 		if (mode === Mode.OPEN) {
-			return this.runOpen(context);
+			this.runOpen(context);
+
+			return true;
+		}
```

Comment:
does tslint not have grammar rules like eslint?  would be nice to apply these, so I can follow code guidelines better :)